core/aws_ddk_core/stages/s3_event.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         event_names: List[str],
         bucket_name: str,
         key_prefix: Optional[str] = None,
+        create_trail: bool = True,
```

Comment:
So now should we add an escape hatch?

Reply:
For the trail you mean? Nop, not possible: https://github.com/aws/aws-cdk/issues/15462#issuecomment-881287691

Reply:
That being said, if the Trail is defined in the same stack, then the user could pass a Trail here and it would work. So perhaps we can use that for consistency?